src/components/Profile/ProfileLayout.tsx

```diff
@@ -45,16 +52,34 @@ export function ProfileLayout({
 
   return (
     <>
+      {user && stats ? (
+        <Meta
+          title={`${user.username} Creator Profile | Civitai`}
+          description={`Average Rating: ${stats.ratingAllTime.toFixed(1)} (${abbreviateNumber(
+            stats.ratingCountAllTime
+          )}), Models Uploaded: ${abbreviateNumber(0)}, Followers: ${abbreviateNumber(
+            stats.followerCountAllTime
+          )}, Total Likes Received: ${abbreviateNumber(
+            stats.favoriteCountAllTime
+          )}, Total Downloads Received: ${abbreviateNumber(stats.downloadCountAllTime)}. `}
+          image={!user.image ? undefined : getEdgeUrl(user.image, { width: 1200 })}
+          links={[{ href: `${env.NEXT_PUBLIC_BASE_URL}/user/${username}`, rel: 'canonical' }]}
```

Comment:
Technically, it would be like this now, right?
```suggestion
          links={[{ href: `${env.NEXT_PUBLIC_BASE_URL}/user/${username}/profile`, rel: 'canonical' }]}
```

Reply:
We have moved away from '/profile'